<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrameDialog</class>
 <widget class="QDialog" name="FrameDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>616</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="frame_properties">
      <attribute name="title">
       <string>Properties</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::ExpandingFieldsGrow</enum>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="projectLabel">
         <property name="text">
          <string>&amp;Project:</string>
         </property>
         <property name="buddy">
          <cstring>_projectBox</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="ProjectTreeComboBox" name="_projectBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="beginLabel">
         <property name="text">
          <string>&amp;Begin:</string>
         </property>
         <property name="buddy">
          <cstring>_beginEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDateTimeEdit" name="_beginEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="currentSection">
          <enum>QDateTimeEdit::HourSection</enum>
         </property>
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="endLabel">
         <property name="text">
          <string>&amp;End:</string>
         </property>
         <property name="buddy">
          <cstring>_endEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QDateTimeEdit" name="_endEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="currentSection">
          <enum>QDateTimeEdit::HourSection</enum>
         </property>
         <property name="calendarPopup">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="notesLabel">
         <property name="text">
          <string>&amp;Notes:</string>
         </property>
         <property name="buddy">
          <cstring>_notesEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QPlainTextEdit" name="_notesEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>1</verstretch>
          </sizepolicy>
         </property>
         <property name="focusPolicy">
          <enum>Qt::StrongFocus</enum>
         </property>
         <property name="placeholderText">
          <string>Notes about this time entry</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>&amp;Archived:</string>
         </property>
         <property name="buddy">
          <cstring>_archivedCheckBox</cstring>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QCheckBox" name="_archivedCheckBox">
         <property name="focusPolicy">
          <enum>Qt::StrongFocus</enum>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="_buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Reset|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ProjectTreeComboBox</class>
   <extends>QComboBox</extends>
   <header>commonWidgets/ProjectTreeComboBox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>_projectBox</tabstop>
  <tabstop>_beginEdit</tabstop>
  <tabstop>_endEdit</tabstop>
  <tabstop>_archivedCheckBox</tabstop>
  <tabstop>_notesEdit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>_buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>FrameDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>FrameDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
